docs/po/pt.po

```diff
@@ -1590,6 +1590,1003 @@ msgstr "`/blockhash/<ALTURA>`: hash do bloco em uma determinada altura de bloco.
+
+#: src/runes.md:146 src/runes/specification.md:378
+msgid "Cenotaphs"
+msgstr "Cenotáfios"
+
+#: src/runes.md:149
+msgid "Runestones may be malformed for a number of reasons, including non-pushdata opcodes in the runestone `OP_RETURN`, invalid varints, or unrecognized runestone fields."
+msgstr "As runestones podem ser mal formadas por várias razões, incluindo “non-pushdata opcodes” na runestone `OP_RETURN`, variantes inválidos, ou campos de runestone não reconhecidos."
+
+#: src/runes.md:153
+msgid "Malformed runestones are termed [cenotaphs](https://en.wikipedia.org/wiki/Cenotaph)."
+msgstr "As runes malformadas são designadas por [cenotáfios](https://es.wikipedia.org/wiki/Cenotafio)."
+
+#: src/runes.md:156
+msgid ""
+"Runes input to a transaction with a cenotaph are burned. Runes etched in a transaction with a cenotaph are set as unmintable. Mints in a transaction with a cenotaph count towards "
+"the mint cap, but the minted runes are burned."
+msgstr ""
+"As runes introduzidas numa transação com um cenotáfio são queimadas. As runes gravadas numa transação com um cenotáfio são consideradas não-mintáveis. Os mints numa "
+"transação com um cenotáfio contam para o limite de mints, mas as runes mintadas são queimadas."
+
+#: src/runes.md:160
+msgid ""
+"Cenotaphs are an upgrade mechanism, allowing runestones to be given new semantics that change how runes are created and transferred, while not misleading unupgraded clients as to "
+"the location of those runes, as unupgraded clients will see those runes as having been burned."
+msgstr ""
+"Os cenotáfios são um mecanismo de atualização, permitindo que as runestones recebam novas semânticas que alteram a forma de como as runes são criadas e transferidas, sem "
+"enganar os clientes não atualizados quanto à localização dessas runes, uma vez que os clientes não atualizados verão essas runes como tendo sido queimadas."
+
+#: src/runes/specification.md:1
+msgid "Runes Does Not Have a Specification"
+msgstr "As Runes não têm uma Especificação"
+
+#: src/runes/specification.md:4
+msgid "The Runes reference implementation, `ord`, is the normative specification of the Runes protocol."
+msgstr "`ord` é a implementação de referência das Runes e constitui a especificação normativa do protocolo de Runes."
+
+#: src/runes/specification.md:7
+msgid ""
+"Nothing you read here or elsewhere, aside from the code of `ord`, is a specification. This prose description of the runes protocol is provided as a guide to the behavior of "
+"`ord`, and the code of `ord` itself should always be consulted to confirm the correctness of any prose description."
+msgstr ""
+"Nada do que ler aqui ou noutro lugar, para além do código `ord`, é uma especificação. Esta descrição em prosa do protocolo runes é "
+"fornecida como um guia para o comportamento do `ord`, e o código do próprio `ord` deve ser sempre consultado para confirmar a correção de qualquer descrição em prosa do mesmo."
```

Comment:
Update this part:
deve ser sempre consultado para confirmar a **exatidão** de qualquer descrição em prosa do mesmo.